doc/ruleSets/PER-CS2.0.rst

```diff
@@ -20,3 +20,7 @@ Rules
 
 - `method_argument_space <./../rules/function_notation/method_argument_space.rst>`_
 - `single_line_empty_body <./../rules/basic/single_line_empty_body.rst>`_
+- `trailing_comma_in_multiline <./../rules/control_structure/trailing_comma_in_multiline.rst>`_ with config:
+
+  ``['after_heredoc' => true, 'elements' => ['arguments', 'arrays', 'match', 'parameters']]``
```

Comment:
this is super confusing for me. ie, on PHP <8, the rule is not configured with `parameters`

Reply:
The fact that some elements are not enabled on PHP 7.4 is implicit and could be seen as an implementation detail of the ruleset. I agree it would be better to make it explicit in the doc somehow, but to me it's not a requirement to merge this PR.